lte/gateway/c/core/oai/tasks/amf/amf_app_pdu_resource_setup_req_rel.cpp

```diff
@@ -52,30 +53,11 @@ uint64_t get_bit_rate(uint8_t ambr_unit) {
 void ambr_calculation_pdu_session(
     std::shared_ptr<smf_context_t> smf_context, uint64_t* dl_pdu_ambr,
     uint64_t* ul_pdu_ambr) {
-  if ((smf_context->dl_ambr_unit == 0) || (smf_context->ul_ambr_unit == 0) ||
-      (smf_context->dl_session_ambr == 0) ||
-      (smf_context->dl_session_ambr == 0)) {
-    // AMBR has not been populated till now and default assigned
-    *dl_pdu_ambr = (64 * 32768);
-    *ul_pdu_ambr = (64 * 32768);
-  } else {
-    // refer 24-501 9.11.4.14
-    if ((smf_context->dl_ambr_unit) < 4) {
-      *dl_pdu_ambr =
-          4 ^ (smf_context->dl_ambr_unit - 1) * (smf_context->dl_session_ambr);
-    } else {
-      *dl_pdu_ambr = smf_context->dl_session_ambr *
-                     get_bit_rate(smf_context->dl_ambr_unit);
-    }
+  *dl_pdu_ambr =
+      smf_context->dl_session_ambr * get_bit_rate(smf_context->dl_ambr_unit);
 
-    if ((smf_context->ul_ambr_unit) < 4) {
-      *ul_pdu_ambr =
-          4 ^ (smf_context->ul_ambr_unit - 1) * (smf_context->ul_session_ambr);
-    } else {
-      *ul_pdu_ambr = smf_context->ul_session_ambr *
-                     get_bit_rate(smf_context->ul_ambr_unit);
-    }
-  }
+  *ul_pdu_ambr =
+      smf_context->ul_session_ambr * get_bit_rate(smf_context->ul_ambr_unit);
```

Comment:
which unit test is covering these?

Reply:
I'll add unit test for ambr_calculation_pdu_session().

Reply:
@ulaskozat, Unit tests are added, please review.